pkg/drivers/kic/kic.go

```diff
@@ -102,8 +103,14 @@ func (d *Driver) Create() error {
 		params.IP = ip.String()
 	}
 	drv := d.DriverName()
+
 	listAddr := oci.DefaultBindIPV4
-	if oci.IsExternalDaemonHost(drv) {
+	if d.NodeConfig.ListenAddress != "" && d.NodeConfig.ListenAddress != listAddr {
+		out.Step(style.Tip, "Minikube is not meant for production use. You are opening non-local traffic")
```

Comment:
thank you for addressing the review comments just on last thing:
minikube should always be lower-case, after all we are "mini" :) lets make this "minikube"

Reply:
Have amended the message and updated the description.